<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatverIniOptimizer</class>
 <widget class="QDialog" name="CatverIniOptimizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Optimize catver.ini</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="toolTip">
      <string>Optimizer log</string>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="toolTip">
      <string>Optimizer status / progress</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>Idle</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonOptimize">
     <property name="toolTip">
      <string>Optimize catver.ini now</string>
     </property>
     <property name="text">
      <string>Optimize</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="toolTip">
      <string>Close this dialog</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>pushButtonOptimize</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>CatverIniOptimizer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
